### PR TITLE
Update generate_solve_rllib_test.py

### PR DESCRIPTION
**Pull request by arashHaratian**
Fix the problems with instantiation of numeric Skd and state representation.

### CODE REVIEW

File: generate_solve_rllib_test.py
Comment (fteicht) [inline generate_solve_rllib_test.py:244]: I think it does not change anything, since val == fluent[args], right?

Comment (fteicht) [reply generate_solve_rllib_test.py:244]: Oh Ok I see! The `enumerate()` operator on `pddl_state.fluents` messed up things in my mind. The code works but does indeed different things. Thanks!

File: generate_solve_rllib_test.py
Comment (fteicht) [inline generate_solve_rllib_test.py]: This will result in a Python exception since the `SkdPPDDLDomain` does not support numeric encoding. Can you please provide a way to write an error message and to gracefully exit the script when the user selects the numeric encoding and the ppddl model options at the same time? Thanks

**Review by fteicht — CHANGES_REQUESTED** [review]

File: generate_solve_rllib_test.py
Comment (arashHaratian) [reply generate_solve_rllib_test.py]: Yes! I did not notice that as I did not try that PPDDL case yet :)
I added a condition, to show an error and quit. see if that suffice.

File: generate_solve_rllib_test.py
Comment (fteicht) [inline generate_solve_rllib_test.py]: Shouldn't it be `and not classic`? @arashHaratian

**Review by fteicht — CHANGES_REQUESTED** [review]

File: generate_solve_rllib_test.py
Comment (arashHaratian) [reply generate_solve_rllib_test.py]: Yes! Sorry it is my mistake. fixed it now X)

**Comment by amathsow** [comment]
Hello @fteicht, 

when we run the script with probablistic flag , specially with arrivals , we have the issue
'AttributeError: SkdSPDDLDomain object has no attribute task'. Can you please double check?

**Review by fteicht — APPROVED** [review]
Thanks @arashHaratian !

**Review by fteicht — APPROVED** [review]

**Review by fteicht — APPROVED** [review]

**Review by fteicht — APPROVED** [review]

**Review by fteicht — APPROVED** [review]

**Comment by amathsow** [comment]
> Hello @fteicht,
> 
> when we run the script with probablistic flag , specially with arrivals , we have the issue 'AttributeError: SkdSPDDLDomain object has no attribute task'. Can you please double check?

solved: need to call state reset to enable task when we use arrivals flag: add self.skd_beluga_domain._state_reset() in the constructor

**Comment by fteicht** [comment]
> when we run the script with probablistic flag , specially with arrivals , we have the issue 'AttributeError: SkdSPDDLDomain object has no attribute task'. Can you please double check?

Hi @amathsow and thanks for reporting!
I've created corresponding issue #3 and created pull request #4 that solves issue #3 .
I'm waiting for an approved review by either @r-eifler or @lompabo to merge the PR in the `main-competition` branch.

**Comment by fteicht** [comment]
Hi @amathsow the issue has been fixed and merged in the `main-competition` branch. Thanks again for reporting.

**Pull request merged**
